examples/pyaudio_oscope_local.py

```diff
@@ -54,3 +54,7 @@
     import sys
     if sys.flags.interactive == 0:
         app.exec_()
+
+# stop and close pyaudio node to properly close pyaudio
```

Comment:
If this is necesaary, then perhaps the device should handle it with an `atexit` callback?